src/auth/AuthSelectors.js

```diff
@@ -0,0 +1,58 @@
+import {createSelector} from 'reselect';
+
+const isLogged = state => state.authReducer.logged;
+const tokenId = state => state.authReducer.tokenId;
+const tokenExpires = state => state.authReducer.tokenExpires;
+const tenant = state => state.authReducer.tenant;
+const tenants = state => state.authReducer.tenants;
+const user = state => state.authReducer.user;
+const inProgress = state => state.authReducer.inProgress;
+
+export const getLoggedState = createSelector(
+  [isLogged],
+  isLogged => {
+    return isLogged;
+  }
+);
+
+export const getTokenId = createSelector(
+  [tokenId],
+  tokenId => {
+    return tokenId;
+  }
+);
+
+export const getTokenExpires = createSelector(
+  [tokenExpires],
+  tokenExpires => {
+    return tokenExpires;
+  }
+);
+
+export const getTenant = createSelector(
+  [tenant],
+  tenant => {
+    return tenant;
+  }
+);
+
+export const getTenants = createSelector(
+  [tenants],
+  tenants => {
+    return tenants;
+  }
+);
+
+export const getUser = createSelector(
+  [user],
+  user => {
+    return user;
+  }
+);
+
+export const getProgressState = createSelector(
+  [inProgress],
+  inProgress => {
+    return inProgress;
```

Comment:
Maybe just using `argument => argument` in each "oneliner" in this file? :smile: